### ISSUE TITLE
Nether Wart Block Not Compiling As Leaves

### ISSUE THREAD

**Issue by TheDarkPanther** [PandaMods-Dev/Pandas-Falling-Trees#96]
I was trying to get nether trees to work through the whitelist/blacklist system I made sure to put all of it in the correct way and nether wart block doesn't seem to work but I got warped wart block to work.

Video of the problem:
https://github.com/user-attachments/assets/78588fc4-a4f1-4213-bc2b-e60133fb47af

My Config file:
[fallingtrees_common.json](https://github.com/user-attachments/files/17283805/fallingtrees_common.json)

**Comment by mkramer541** [comment]
I'm not sure if you ever resolved this, but I noticed you had "block" spelled wrong on line 35 in your config file. Unrelated, but I think line 38 should have the tag as "minecraft:shroomlight".

**Comment by Skulgan** [comment]
Politely and respectfully submitting (pleading and bawling and on my knees) to resolve this issue and put it at the top of whatever priority list you may have for bugfixes involving this mod.

**Comment by TheDarkPanther** [comment]
> I'm not sure if you ever resolved this, but I noticed you had "block" spelled wrong on line 35 in your config file. Unrelated, but I think line 38 should have the tag as "minecraft:shroomlight".

We literally spelt it the same way...I am confused... 

Edit: I see it now lmao...

**Comment by Delphilious-Skeleton** [comment]
Right, I have spent the last three hours trying to diagnose this issue and see if I can't get it working. But I've come to the same conclusion the rest of these great folk have and I've come to spell out what I've learned. Let me preface, I am by no means a coder, I just used some trial and error along with Claude to see if I could get something working.

TLDR; No data pack or changing of the common.json file to include the stems and associated blocks worked

I attempted to create a data pack (at the behest of first Gemini) and see if the mod simply lacked the ability to read certain blocks. I was successful in getting the data pack to be read, but it still failed to make the trees act in accordance to the mod. 
I tried then to see if editing the common.json would work at all, and the same result occurred. 
Despite all of that, I did manage to get the data pack to recognize the stems and warped blocks (data pack attached if anyone wants it).

I'm not well versed in understanding how mods work or why this bug exists. But I wanted to be helpful in some way so that maybe someone who also is struggling with this can see that someone else had the same thought and I can save them some time.

EDIT:
I had Claude summarize the process and made a bug report which I added alongside the data pack

[bug-report.md](https://github.com/user-attachments/files/29583741/bug-report.md)

[NetherTreeCompat.zip](https://github.com/user-attachments/files/29583624/NetherTreeCompat.zip)
